src/app/static/src/js/components/runReportTabs/runningReportsDetails.vue

```diff
@@ -0,0 +1,185 @@
+<template>
+    <div>
+        <div class="container">
+            <div id="report-log">
+                <div class="row pt-2">
+                    <div id="report-git-branch" v-if="reportLog.git_branch" class="col-sm-auto">
+                        <div class="text-right">
+                            <span>Github branch:</span>
+                            <span class="font-weight-bold">
+                            {{ reportLog.git_branch }}
+                        </span>
+                        </div>
+                    </div>
+                    <div id="report-git-commit" v-if="reportLog.git_commit" class="col-sm-auto">
+                        <div class="text-right">
+                            <span>Github commit:</span>
+                            <span class="font-weight-bold">
+                            {{ reportLog.git_commit }}
+                        </span>
+                        </div>
+                    </div>
+                </div>
+                <div class="row pt-2">
+                    <div id="report-params" v-if="paramSize > 0" class="col-sm-auto">
+                        <span>Parameters:</span>
+                        <span>
+                        <div class="d-md-table-row row" v-for="(n, index) in paramSize">
+                            <span class="border border-secondary p-1">{{ reportLog.params[index].name }}:</span>
+                            <span class="border border-secondary p-1">
+                                {{ reportLog.params[index].value }}
+                            </span>
+                        </div>
+                    </span>
+                    </div>
+                    <div id="report-database-source" v-if="instanceSize > 0" class="col-sm-auto">
+                        <span>Database(source):</span>
+                        <span>
+                        <ul class="d-md-table-row list-unstyled" v-for="(n, index) in instanceSize">
+                            <li class="p-1 font-weight-bold">{{ reportLog.instances[index].source }}</li>
```

Comment:
'Source' and 'Annexe' can't be hardcoded here, they are just the names of the instances in the example in the mockup.